coalib/results/Diff.py

```diff
@@ -276,23 +282,36 @@ def split_diff(self, distance=1):
         # always.
         yield this_diff
 
-    def range(self, filename):
+    def range(self, filename=None):
         """
         Calculates a SourceRange spanning over the whole Diff. If something is
         added after the 0th line (i.e. before the first line) the first line
         will be included in the SourceRange.
 
         The range of an empty diff will only affect the filename:
 
+        >>> range = Diff([], filename="file").range()
+        >>> range.file is None
```

Comment:
Re-worked without the print, I had to give it an explicit filename, `/file`, to avoid it automatically prepending the working directory